starctl-service-rest

```diff
@@ -40,7 +40,7 @@ else
     fi
 fi
 
-eval declare -a JAVA_OPTS=("$JAVA_OPTS")
+eval declare -a JAVA_OPTS=("$REST_JAVA_OPTS")
```

Comment:
Should similar change be made to `starctl-service-rest`, to allow different JAVA_OPTS for different services?
(and/or should there be both `JAVA_OPTS` and service-specific ones?)

Reply:
This is `starctl-service-rest`.
I made the change because that's the way it was done in the c2 repo.

Reply:
Whops. Sorry about noise, yes, you are right. That change makes sense and I somehow read this to be for graphql one.